sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/samples/simple-chat/java/src/main/java/com/functions/Functions.java

```diff
@@ -0,0 +1,72 @@
+package com.functions;
+
+import java.util.*;
+import com.microsoft.azure.functions.annotation.*;
+import com.microsoft.azure.functions.*;
+import com.microsoft.azure.functions.signalr.annotation.*;
+import com.microsoft.azure.functions.signalr.*;
+
+public class Functions {
```

Comment:
@weshaggard is it alright to have Java samples in the repo? They aren't expected to build (at least at this point we are not building samples).

Reply:
I would suggest we do not have java samples in the .NET repo. I assume these samples would need to to be calling java libraries so they should live in the java repo with the java library.

Reply:
> I assume these samples would need to to be calling java libraries so they should live in the java repo with the java library.

There aren't really Java equivalent libraries to call - the .NET code is used via an out of process mechanism.

Reply:
I think we could move the Java, Js sample codes to https://github.com/aspnet/AzureSignalR-samples/